files/project-templates/app/Makefile

```diff
@@ -26,17 +20,15 @@ PROJECT_NAME := {{ projectName }}
 install:
 	$(HIDE)npm install
 	$(HIDE)cd node_modules/karma-jasmine-jquery && node install.js
-	$(HIDE)node node_modules/.bin/webdriver-manager update --standalone
 
 clean:
 	$(HIDE)rm -rf bundle coverage
 
-test: webpack-test
+test: karma-test
 
-coverage: webpack-coverage
+coverage: karma-coverage
 
-report-coverage:
-	$(HIDE)echo "Reporting Coverage not implemented yet"
+report-coverage: karma-coveralls
```

Comment:
Eh, I'm not really sure, it seemed better than a placeholder, coveralls is free for open source projects.